packages/jest-util/src/create_process_object.js

```diff
@@ -22,49 +22,52 @@ function createProcessEnv() {
   const real = Object.create(proto);
   const lookup = {};
 
-  const proxy = new Proxy(real, {
-    deleteProperty(target, key) {
-      for (const name in real) {
-        if (real.hasOwnProperty(name)) {
-          if (typeof key === 'string' && process.platform === 'win32') {
-            if (name.toLowerCase() === key.toLowerCase()) {
-              delete real[name];
-              delete lookup[name.toLowerCase()];
-            }
-          } else {
-            if (key === name) {
-              delete real[name];
-              delete lookup[name];
+  if (typeof Proxy === 'undefined') {
+    return real;
```

Comment:
`real` is an empty object here, since it's just created based on the prototype. A better fix would be to do, just after the signature method:

```javascript
if (typeof Proxy === 'undefined') {
  return deepCyclicCopy(process.env);
}
```

Reply:
Sure. Wait a second. Do you have some `deepCyclicCopy` method already in the project?